katas/content/distinguishing_states/two_orthogonal_three_qubit/solution.md

```diff
@@ -0,0 +1,27 @@
+Let's find a unitary transformation that converts the state $\ket{S_0}$ to the basis state $\ket{000}$. To do this, we first apply a unitary operation that maps the first state to the W state 
+$\frac{1}{\sqrt{3}} \big(\ket{100} + \ket{010} + \ket{001}\big)$.
+
+> We will use a convenient rotation gate $R_1$ which applies a relative phase to the $\ket{1}$ state and doesn't change the $\ket{0}$ state.
+> In matrix form: 
+>
+> $$R_1(\theta) = \begin{bmatrix} 1 & 0 \\ 0 & e^{i\theta} \end{bmatrix} $$
```

Comment:
This reads a bit oddly as a note, as if we're explaining how we prepare W state rather than what happens next. Let's rephrase
```suggestion
After this, we will use a convenient rotation gate $R_1$ which applies a relative phase to the $\ket{1}$ state and doesn't change the $\ket{0}$ state.
In matrix form: 

$$R_1(\theta) = \begin{bmatrix} 1 & 0 \\ 0 & e^{i\theta} \end{bmatrix} $$
```